Extension/package.json

```diff
@@ -503,7 +503,7 @@
             "Enabled",
             "Disabled"
           ],
-          "default": "Disabled",
+          "default": "Enabled",
```

Comment:
You should also update `cpptools.json` or else this will be immediately overridden by the A/B testing logic.